config/config.exs

```diff
@@ -66,7 +66,29 @@ config :notesclub, Notesclub.Scheduler,
       task: {Notesclub.Searches.Populate, :next, []}
     ]
   ]
-
+if System.get_env("IS_OBAN_WEB_PRO_ENABLED") do
```

Comment:
Doubtful that anybody would have `IS_OBAN_WEB_PRO_ENABLED=0` or something, but you're explicitly checking for `true` elsewhere:

```suggestion
if System.get_env("IS_OBAN_WEB_PRO_ENABLED") == "true" do
```

Reply:
Btw, thanks for the suggestion. I couldn't accept it there as the code was outdated.